trigger/version.go

```diff
@@ -193,7 +193,23 @@ func (t *versionTriggers) versionGithubCheckOutcome(sub *event.Subscription) (*n
 }
 
 func (t *versionTriggers) versionFailure(sub *event.Subscription) (*notification.Notification, error) {
-	if t.data.Status != evergreen.VersionFailed {
+	versionTasks, err := task.FindAll(task.ByVersion(t.version.Id))
```

Comment:
If we query by Version ID and Aborted == true, then we can use task.Count() and just check if count > 0, rather than iterating through all tasks (should put this query in the task/db.go file)

Reply:
Given your comment on the above thread we actually can't do this (alas), disregard

Reply:
Sure I think this would work if we also check that the number of aborted tasks equals the number of failed tasks.

Reply:
True, although this would require two queries which I don't think is an improvement. We could filter by version and the failed status though to limit how many tasks we have to iterate through.